eq.yml

```diff
@@ -87,6 +87,7 @@ services:
     image: onsdigital/eq-author-api
     environment:
       DB_CONNECTION_URI: postgres://postgres:mysecretpassword@eq-author-db:5432/postgres
+      NODE_ENV: production
```

Comment:
I think it would be better to build the containers with these set inside them.

Reply:
Yeah, good shout. It would prevent having to raise yet another PR for the deploy to terraform scripts. 👍

Reply:
@jonnyshaw89 Raised a PR on Author API, https://github.com/ONSdigital/eq-author-api/pull/50. Will close this PR.